docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,150 @@
+# Version 2.15.0 (March 2024)
+
+Welcome to the Zowe Version 2.15.0 release!
+
+See [New features and enhancements](#new-features-and-enhancements) for a full list of changes to the functionality. See [Bug fixes](#bug-fixes) for a list of issues addressed in this release.
+
+**Download v2.15.0 build**: Want to try new features as soon as possible? You can download the v2.15.0 build from [Zowe.org](https://www.zowe.org/download.html).
+
+## New features and enhancements
+
+Zowe Version 2.15.0 contains the enhancements that are described in the following topics.
+
+:::info find out more
+To watch a demo of new enhancements and updated features included in a Zowe minor release, look for the release demo recording in the [Zowe V2 System Demo playlist](https://www.youtube.com/playlist?list=PL8REpLGaY9QGjSTAqZaWxLG_g-jW1qGmo) on YouTube.
+
+System demos are typically held the week after a minor release becomes available. Check the [Open Mainframe Project Calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar) for the latest schedule.
+:::
+
+### Zowe installation and packaging
+
+### Zowe Application Framework
+
+#### Zlux App Server
+
+
+### Zowe API Mediation Layer
+
+- Timeout configuration values of the websocket webclient are now supported in the API Gateway. ([#3315](https://github.com/zowe/api-layer/issues/3315))
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Added `--binary` and `--encoding` options to the `zos-files edit` command in Zowe CLI V2.
```

Comment:
@adam-wolfe - would like to explain the benefit of adding these two options.

Reply:
These options allow users to download and edit binary files and data sets or to edit files and data sets with a user-specified encoding.

Reply:
Issue: https://github.com/zowe/zowe-cli/issues/1725